packages/dashboard/src/app/api/useStoryApi.js

```diff
@@ -184,15 +177,19 @@ const useStoryApi = () => {
         });
       }
     },
-    [createStoryFromTemplateCallback]
+    [apiCallbacks]
   );
 
+  if (!apiCallbacks.createStoryFromTemplate) {
+    createStoryFromTemplate = noop;
+  }
```

Comment:
Why is this needed?

I'd tend to think that `createStoryFromTemplate` should never be called in the first place when the API callback isn't provided, because any button to create a story from a template would not be visible.

Reply:
Not needed, it was only for handling some future case influenced by how the function is being called inside `useEffect` here.

https://github.com/google/web-stories-wp/blob/5b713a3f908660330014bfbac951c95d23f092df/packages/dashboard/src/app/views/myStories/index.js#L106-L112

Removed in https://github.com/google/web-stories-wp/pull/9752/commits/8dce4460fc58a70786f8eac72ab86672636925fc